listenbrainz/tests/integration/test_timescale_writer.py

```diff
@@ -17,10 +18,11 @@
 from datetime import datetime
 
 
-class TimescaleWriterTestCase(IntegrationTestCase):
+class TimescaleWriterTestCase(IntegrationTestCase, TimescaleTestCase):
 
     def setUp(self):
-        super(TimescaleWriterTestCase, self).setUp()
+        IntegrationTestCase.setUp(self)
+        TimescaleTestCase.setUp(self)
```

Comment:
Since now we are using user_ids instead of user names and user ids are `SERIAL`, we  get conflict in various tests. We disucssed in IRC and came up with the solution to make a util method which assigns random user ids and use it everywhere. Since, that is a wholesale change to make everywhere I'd prefer to do it in a separate PR as part of general improvements to tests.

Reply:
:+1: